### ISSUE TITLE
Region landmark errors

### ISSUE THREAD

**Issue by webdigga** [component-driven/cypress-axe#135]
A few days ago I started to receive violations from our accessibility tests.  The code for the markup has not changed so I am not sure why I all of a sudden started to receive these errors.

I was using version 0.12.2.

Here is a screenshot of the errors I am now receiving:-

<img width="472" alt="Screenshot 2022-06-21 at 11 34 43" src="https://user-images.githubusercontent.com/571319/174780460-a427fedb-696d-49b2-9e82-af76b9b4699d.png">

I can fix the 2 HTML elements so that they are in more appropriate places in the markup.
But the <esi> tags (which have never sat inside any landmark HTML tag), do not seem to be able to be ignored..

So I have 2 questions.

1. Was there some update a few days ago that might have introduced these new failures for me?
2. Is there any way to ignore the <esi> tags (Akamai Edge side includes)?

Thanks
David
